rdmd_test.d

```diff
@@ -347,15 +354,20 @@ void runTests()
     assert(res.status == 0, res.output);
     assert(!res.output.canFind("compile_force_src"));
 
-    auto fullCompilerPath = environment["PATH"]
-        .splitter(pathSeparator)
-        .map!(dir => dir.buildPath(compiler ~ binExt))
-        .filter!exists
-        .front;
+    // for absolute compiler path, we make the path relative again
+    string fullCompilerPath = void;
+    if (compiler.isAbsolute)
+        fullCompilerPath = compiler.asRelativePath(getcwd()).array;
```

Comment:
Wait, why is this branch computing a relative path when the other branch is computing an absolute one?